### ISSUE TITLE
Is there a xcode project for this project

### ISSUE THREAD

**Issue by ghost** [silento/Location#3]
Is there a xcode project for this project

I can only see couple of files and no project is available which I can execute directly using XCode ?

**Comment by glennposadas** [comment]
No. You have to make your own project and just implement and use this library.
